tests/TestCase/Form/FormTest.php

```diff
@@ -140,16 +140,27 @@ public function testValidateDeprected()
 
             $data = [];
             $this->assertFalse($form->validate($data));
-            $this->assertCount(1, $form->errors());
+            $this->assertCount(1, $form->getErrors());
         });
     }
 
     /**
      * Test the errors methods.
      *
      * @return void
+     * @deprecated 3.7.0 Use Form::testGetErrors() instead.
      */
     public function testErrors()
+    {
+        $this->testGetErrors();
```

Comment:
Why deprecate test method?

`testErrors()` should be removed now as there is new `testGetErrors()`

Now `testGetErrors()` runs twice

Reply:
Would you mind fixing it please :)